test/1.4/legacy/T_c_vect_without_provisional_enabled.dml

```diff
@@ -1,12 +1,13 @@
 /*
-  © 2021 Intel Corporation
+  © 2024 Intel Corporation
   SPDX-License-Identifier: MPL-2.0
 */
-dml 1.4;
 
-/// COMPILE-ONLY
+dml 1.4;
 
 device test;
 
+/// COMPILE-ONLY
+/// DMLC-FLAG --simics-api=7
 /// WARNING WEXPERIMENTAL
-session int vect x;
```

Comment:
With the 1.2 tests refactored to use the provisional feature, this means there is one case left untested: that the compat feature is on, and that it _doesn't_ report `WEXPERIMENTAL` for 1.2. I would typically view this as optional (the dmlparse code speaks for itself), but in this case it presents a regression in coverage.

Reply:
I added `1.2/errors/T_ECVECT` to compensate for this (somewhat awkwardly named since it doesn't actually trigger ECVECT).